### PR TITLE
Change message.reply to add discord embed to clean up reply

### PR DESCRIPTION
**Pull request by Jykinturah**
This should work theoretically but should be tested.
Screen Shot 2019-06-26 at 9 24 18 AM

This does not fix the problem where the bot may pull a deviantArt Eclipse version of the website, breaking the way getMETA works to fetch the image and other metadata using regex.

### CODE REVIEW

File: functions/deviantArt.js [hunk 1]
```diff
@@ -35,7 +35,22 @@ module.exports = {
         "*Hmm... not bad. Not bad at all.*"
       ];
 
-      message.reply(chance.pickone(responseList) + " " + meta.image);
+      // Old reply puts entire link into message
+      // message.reply(chance.pickone(responseList) + " " + meta.image);
```
Comment (tsitra) [inline functions/deviantArt.js]: Delete, don't comment out.

File: functions/deviantArt.js [hunk 2]
```diff
@@ -35,7 +35,22 @@ module.exports = {
...
+
+      /* New reply puts image url into a discord embed object which allows hiding the link
+         in a nicer looking box. Documentation linked below:
+         https://discordjs.guide/popular-topics/embeds.html#using-an-embed-object
+       */
```
Comment (tsitra) [inline functions/deviantArt.js]: A+ documentation

File: functions/deviantArt.js [hunk 3]
```diff
@@ -35,7 +35,22 @@ module.exports = {
...
+
+      message.reply(chance.pickone(responseList), 
+        { embed: {
+          title: meta.title,
+          url: meta.url,
+          description: meta.description,
+          image: { url: meta.image }
+        }
+      });
```
Comment (tsitra) [inline functions/deviantArt.js]: <nitpick> This JSON could be pretty-formatted a little better

Comment (tsitra) [reply functions/deviantArt.js]:
```javascript
{
  embed: {
    title: meta.title,
    url: meta.url,
    description: meta.description,
    image: { 
      url: meta.image 
    }
  }
}
```

**Comment by Jykinturah** [comment]
npm is freaking out about something, which is odd. I didn't change anything other than deviantArt.js very deliberately. There should be no deltas on any other files. I've never used Heroku so I can't begin to guess what's up.

**Comment by tsitra** [comment]
Please PR into `feature-da-link-update` instead of `master` so I can better poke at it from there.

https://github.com/novels-nook/philomena/tree/feature-da-link-update

**Comment by tsitra** [comment]
Actually, I'm going to make a fork and fix it there. But your merge depends on my merge first.

**Pull request closed without merging**